af_error_reporting/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Af Error Reporting",
+    "summary": "Arbetsförmedlingen error reporting for CRM",
+    "version": "12.0.0.1.0",
+    "category": "Utility",
+    "description": """
```

Comment:
Change description to AF uniform,  see example
    'description': """
Jira
===================================================================== \n
V12.0.0.1.0 AFC-2696: Arbetssokande model added \n
    """,